ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml

```
@@ -152,7 +152,7 @@ tests:
   optional: true
 - as: e2e-aws-ovn
   optional: true
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg(/.+)?|cmd/openshift-install|terraform/providers)/(agent|alibabacloud|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  run_if_changed: (fcos|okd)
```

@patrickdillon should this've been?
```suggestion
  run_if_changed: (fcos|scos|okd)
```

---

Fixing this in https://github.com/openshift/release/pull/45141